app/models/miq_server/environment_management.rb

```diff
@@ -29,16 +29,8 @@ def minimal_env_options
     end
 
     def startup_mode
-      mode = ""
-      # Find out startup mode
-      if self.minimal_env?
-        mode = "Minimal"
-        mode << " [#{minimal_env_options.join(', ')}]" unless minimal_env_options.empty?
-      else
-        mode = "Normal"
-      end
-
-      mode
+      return "Normal" unless minimal_env?
+      "Minimal".tap { |s| s << " [#{minimal_env_options.join(', ')}]" unless minimal_env_options.empty? }
```

Comment:
Can decide if this is better than the original ...

Maybe something using interpolation?
`"Minimal#{" " + minimal_env_options.join(', ') unless minimal_env_options.empty?}"`

Reply:
Or maybe use `if minimal_env_options.present?` rather than unless empty?

Reply:
I like tap + present.  The string interpolation way was harder for me to grok on first read.  🤷‍♂️